core/flyout_base.js

```diff
@@ -514,7 +513,7 @@ Blockly.Flyout.prototype.show = function(flyoutDef) {
 /**
  * Create the contents array and gaps array necessary to create the layout for
  * the flyout.
- * @param {!Array<Blockly.utils.toolbox.FlyoutItem>} parsedContent The array
+ * @param {!Array<Blockly.utils.toolbox.FlyoutItemJson>} parsedContent The array
```

Comment:
You have a type for `ToolboxJson` that's an array of `ToolboxItemJson`. Make a parallel type, `FlyoutJson`, instead of explicitly having this be an array.

Reply:
The ToolboxJson is a an object with contents `{'contents': []}` having FlyoutJson seems a bit misleading since it is actually an array. Do any of these sound ok?`FlyoutJsonArray` or `ContentsArray` or `FlyoutItemsArray`.

Reply:
Hm I think `FlyoutItemJsonArray`--it's longer but most descriptive.

`ContentsArray` would get overloaded too easily I think.

Reply:
Updated